subprojects/docs/src/docs/userguide/publishingIvy.adoc

```diff
@@ -132,111 +86,57 @@ Overriding the default identity values is easy: simply specify the `organisation
-
-You can configure the tasks generated by the Ivy Publish Plugin to be skipped based on certain criteria. The following sample demonstrates how to restrict the `binary` publication to the `external` repository and the `binaryAndSources` publication to the `internal` repository.
-
-++++
-<sample dir="ivy-publish/conditional-publishing" id="publishingIvyConditionally" title="Configuring which artifacts should be published to which repositories">
-    <sourcefile file="build.gradle" snippet="task-config"/>
-    <output args="publish"/>
-</sample>
-++++
-
-Moreover, you may want to define your own shorthand tasks to fit your workflow. The following sample defines two tasks: `publishToExternalRepository` to publish all publications to the `external` repository and `publishToInternalRepository` for the `internal` repository:
-
-++++
-<sample dir="ivy-publish/conditional-publishing" id="shorthandTasks" title="Defining your own shorthand tasks for publishing">
-    <sourcefile file="build.gradle" snippet="shorthand-tasks"/>
-</sample>
-++++
+ * The name of the repository (optional)
+ * Its URL
+ 
+You can define multiple repositories as long as they have unique names within the build script. You may also declare one (and only one) repository without a name. That repository will take on an implicit name of "Ivy".
 
+You can also configure any authentication details that are required to connect to the repository. See api:org.gradle.api.artifacts.repositories.IvyArtifactRepository[] for more details.
 
 [[publishing_ivy:descriptor]]
 === Generating the Ivy module descriptor file without publishing
 
 At times it is useful to generate the Ivy module descriptor file (normally `ivy.xml`) without publishing your module to an Ivy repository. Since descriptor file generation is performed by a separate task, this is very easy to do.
 
-The Ivy Publish Plugin creates one api:org.gradle.api.publish.ivy.tasks.GenerateIvyDescriptor[] task for each registered api:org.gradle.api.publish.ivy.IvyPublication[], named `generateDescriptorFileFor«_PUBNAME_»Publication`, which will be `generateDescriptorFileForIvyJavaPublication` for the previous example of the `ivyJava` publication.
+The Ivy Publish Plugin creates one api:org.gradle.api.publish.ivy.tasks.GenerateIvyDescriptor[] task for each registered api:org.gradle.api.publish.ivy.IvyPublication[], named `generateDescriptorFileFor__PubName__Publication`, which will be `generateDescriptorFileForIvyJavaPublication` for the previous example of the `ivyJava` publication.
 
-You can specify where the generated Ivy file will be located by setting the `destination` property on the generated task. By default this file is written to `build/publications/«_PUBNAME_»/ivy.xml`.
+You can specify where the generated Ivy file will be located by setting the `destination` property on the generated task. By default this file is written to `build/publications/__PubName__/ivy.xml`.
```

Comment:
This sample also requires `enableFeaturePreview("STABLE_PUBLISHING")`.

Reply:
- [ ]  Fix sample _ivy-publish/descriptor-customization_ to use deferred configuration